src/components/pages/weather/elements/WeatherHeadingCell.js

```diff
@@ -2,9 +2,13 @@ import React, { Component } from 'react';
 
 export default class WeatherHeadingCell extends Component {
   render() {
+    const { earthDate } = this.props;
+    const date = new Date(earthDate);
```

Comment:
I think object destructuring is a little confusing when it's just one value - much clearer to say `const earthDate = this.props.earthDate;`.

We could also do this in one line using:

`const date = new Date(this.props.earthDate);`

Reply:
I had "const earthDate = this.props.earthDate;" originally but the formatting asks you to restructure it.
I've changed it to the one line version which is much nicer than what I had before.

Reply:
Thanks for letting me know! I've updated the config to not error on that, but I guess it's good if it's taught you about destructuring!